packs/spell-effects/spell-effect-mantle-of-the-magma-heart.json

```diff
@@ -188,6 +201,23 @@
                 ],
                 "type": "fire",
                 "value": 5
+            },
+            {
+                "key": "Note",
+                "predicate": [
+                    "self:heart-of-fire",
+                    {
+                        "or": [
+                            "item:trait:melee",
+                            {
+                                "not": "item:trait:reach"
+                            }
+                        ]
+                    }
```

Comment:
I think we can lose the or. It's supposed to be active on non-reach melee strikes. This would be active on any melee strike and any strike without reach.